src/odoc/bin/main.ml

```diff
@@ -574,15 +655,38 @@ end = struct
       & opt (some string) None
       & info ~docs ~docv:"PATH.odocl" ~doc [ "o" ])
 
+  let page_pkgnames =
+    let doc =
+      "Specifies a directory PATH containing pages that can be referenced by \
+       {!/pkgname} during linking. A pkgname can be specified in the -P \
```

Comment:
What does "during linking" mean here ? `{!/pkgname}` looks like odoc syntax, which is used during the writing of the doc, not linking.

Reply:
Indeed, they are referenced during the writing of the doc. The reference is resolved at link time, but this might not be relevant.
I'll remove the "during linking".